src/robot/conf/languages.py

```diff
@@ -55,48 +62,77 @@ def _add_language(self, lang):
         self.true_strings |= {s.title() for s in lang.true_strings}
         self.false_strings |= {s.title() for s in lang.false_strings}
 
-    def _get_languages(self, languages):
-        languages = self._resolve_languages(languages)
-        available = self._get_available_languages()
+    def _get_languages(self, languages, add_default=True):
+        languages = self._resolve_languages(languages, add_default)
+        available = self.get_available_languages()
         returned = []
         for lang in languages:
             if isinstance(lang, Language):
                 returned.append(lang)
             else:
+                module, lang_part = lang.rsplit(":", 1) if ":" in lang else (None, lang)
+                if module is not None:
+                    try:
+                        available.update(self.import_languages_module(module))
+                        lang = lang_part
+                    except DataError:
+                        pass
```

Comment:
What's this? Is the idea to support something like `ManyLangs.py:SomeLang` where `ManyLangs` would contain multiple languages and only `SomeLang` should be taken into use? I don't consider that too useful. You can easily have separate language modules for each language if needed. I'm not totally against adding such functionality, but it would then need separate acceptance tests and documentation and that's not worth the effort in my opinion. Let's stick with API enhancements and try to get this done so that we get rc 2 out soon ASAP.